### PR TITLE
Prl 6257

### PR DESCRIPTION
**Pull request by Scralfie**
### Change description

Event: C100 Intenational Element

### JIRA link

[PRL-6257](https://tools.hmcts.net/jira/browse/PRL-6257)

**Before merging a pull request make sure that:**

- [x] Commits are meaningful and simple
- [x] All commits are squashed into a single commit
- [x] README and other documentation has been updated / added (if needed)

### CODE REVIEW

File: e2e/pages/manageCases/createCase/C100/internationalElement/internationalElement1Page.ts [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+        page,
+        `${Selectors.GovukFormLabel}:text-is("${InternationalElement1Content.formLabelNo}")`,
+        3,
+      ),
+    ]);
+    if (accessibilityTest) {
+      await AccessibilityTestHelper.run(page);
+    }
+  }
+  private static async fillInFields({
```
Comment (JakeCG) [inline e2e/pages/manageCases/createCase/C100/internationalElement/internationalElement1Page.ts:86]: Needs a whitespace above

File: e2e/tests/manageCases/createCase/C100/c100InternationalElement.spec.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      accessibilityTest: false,
+      yesNoInternationalElement: true,
+      subJourney: true,
+    });
+  });
+  test(`Complete the C100 Create case International Element as a solicitor with the following options:
+  Not Accessibility testing,
+  Not Error message testing,
+  Saying no to all options,
+  Setting the applicant Gender to male. @crossbrowserManageCases`, async ({
```
Comment (JakeCG) [inline e2e/tests/manageCases/createCase/C100/c100InternationalElement.spec.ts:27]: Remove tag

**Review by JakeCG — CHANGES_REQUESTED** [review]

**Review by OLT2000 — DISMISSED** [review]
lgtm

**Review by JakeCG — APPROVED** [review]

**Review by OLT2000 — APPROVED** [review]
lg

**Pull request merged**